src/OSPSuite.Core/Domain/Builder/PathAndValueEntityBuildingBlockFromPKSim.cs

```diff
@@ -5,6 +5,7 @@ namespace OSPSuite.Core.Domain.Builder
    public abstract class PathAndValueEntityBuildingBlockFromPKSim<T> : PathAndValueEntityBuildingBlock<T> where T : PathAndValueEntity
    {
       public string PKSimVersion { set; get; }
+      public string SnapshotOriginModuleId { set; get; }
```

Comment:
Would it make sense to say that what this is and also explain udner which condition it's null?